float64 value
---
int32 errorCode
string errorDescription